lib/devices/MiioVacuum.js

```diff
@@ -309,18 +365,36 @@ class MiioVacuum {
     }
 
     /**
+     * @typedef AppLocale
+     * @property {string} name
+     * @property {string} bom
```

Comment:
what’s that?

Reply:
I don't know.
Roborock returns 'bom': 'A.03.0005'